programs/utilities/slimechunk.sc

```diff
@@ -0,0 +1,8 @@
+//Run the slimechunk to tell whether you are in a slime chunk
+__command() ->
+(
+    position = query(player(), 'pos');
+    is_slimechunk = in_slime_chunk(position);
+    if (is_slimechunk == 'true', exit('§aSlime chunk!'));
+    if (is_slimechunk == 'false', exit('§cNot a slime chunk :\('));
```

Comment:
```suggestion
    if (is_slimechunk, '§aSlime chunk!', '§cNot a slime chunk :\(');
```

The `in_slime_chunk()` function doesn't return a string of `'true'` or `'false'`, rather it returns a boolean value (that is, the value `true` or `false`.
That means that you can just use the if/else statement to return a message based on whether or not it's a slime chunk.

Also, fyi, the `exit()` function is typically used in case of an error which means you can't continue with execution, not at the end of the function.